charts/k8s-monitoring/README.md

```diff
@@ -327,11 +327,20 @@ The Prometheus and Loki services may be hosted on the same cluster, or remotely
 | prometheus-windows-exporter.config | string | `"collectors:\n  enabled: cpu,cs,container,logical_disk,memory,net,os\ncollector:\n  service:\n    services-where: \"Name='containerd' or Name='kubelet'\""` | Windows Exporter configuration |
 | prometheus-windows-exporter.enabled | bool | `false` | Should this helm chart deploy Windows Exporter to the cluster. Set this to false if your cluster already has Windows Exporter, or if you do not want to scrape metrics from Windows Exporter. |
 | receivers.grpc.disable_debug_metrics | bool | `true` | It removes attributes which could cause high cardinality metrics. For example, attributes with IP addresses and port numbers in metrics about HTTP and gRPC connections will be removed. |
-| receivers.grpc.enabled | bool | `true` | Receive telemetry data over gRPC? |
-| receivers.grpc.port | int | `4317` | Which port to use for the gRPC receiver. This port needs to be opened in the grafana-agent section below. |
+| receivers.grpc.enabled | bool | `true` | Receive OpenTelemetry signals over OTLP/gRPC? |
+| receivers.grpc.port | int | `4317` | Which port to use for the OTLP/gRPC receiver. This port needs to be opened in the grafana-agent section below. |
 | receivers.http.disable_debug_metrics | bool | `true` | It removes attributes which could cause high cardinality metrics. For example, attributes with IP addresses and port numbers in metrics about HTTP and gRPC connections will be removed. |
-| receivers.http.enabled | bool | `true` | Receive telemetry data over HTTP? |
-| receivers.http.port | int | `4318` | Which port to use for the HTTP receiver. This port needs to be opened in the grafana-agent section below. |
+| receivers.http.enabled | bool | `true` | Receive OpenTelemetry signals over OTLP/HTTP? |
```

Comment:
nit:
Comparison to `receivers.jaeger.` options made me think that maybe it would be more self-explanatory if we use

- `receivers.otlp.http.enabled`  instead `receivers.http.enabled`
- `receivers.otlp.http.port` instead `receivers.http.port`
- `receivers.otlp.grpc.enabled`  instead `receivers.grpc.enabled`
- etc...

not in this PR, just suggestion